<?xml version="1.0"?>
<launch>
  <arg name="GLONAS" default="0"/>
  <arg name="GPS" default="1"/>
  <arg name="BEIDOU" default="1"/>
  <arg name="GALILEO" default="1"/>
  <arg name="Surveytime" default="120"/>
  <arg name="Surveyacc" default="50000"/>
  <arg name="message_rate" default="10"/>

	<group ns="base">
		<node name="f9p" pkg="ublox" type="ublox_ros">
			<param name="serial_port" value="/dev/ttyACM0"/>
			<param name="chain_level" value="0"/>
			<param name="base_type" value="moving"/>
			<param name="rover_quantity" value="1"/>
 			<param name="local_host" value="192.168.0.111"/>
			<param name="local_port" value="16140"/>
			<param name="rover_host" value="192.168.0.202"/>
			<param name="rover_port" value="16145"/>
			<param name="GPS" value="$(arg GPS)"/>
      		<param name="GLONAS" value="$(arg GLONAS)"/>
      		<param name="BEIDOU" value="$(arg BEIDOU)"/>
      		<param name="GALILEO" value="$(arg GALILEO)"/>
      		<param name="Surveytime" value="$(arg Surveytime)"/>
      		<param name="Surveyacc" value="$(arg Surveyacc)"/>
			<param name="message_rate" value = "$(arg message_rate"/>
		</node> 
		<node pkg="rosbag" type="record" name="record" output="screen" args="-o ../data/rtk_tests --all"/>
	</group>
</launch>
